src/App.js

```diff
@@ -38,6 +39,8 @@ function MainApp() {
         <Route path='/web-teams' element={<WebTeam />} />
         <Route path='/faculty-teams' element={<FacultyTeam />} />
 
+        <Route path='/send-email' element={<SendEmail/>} />
```

Comment:
This route should be accessible to the admins only. You can move this route component in `<Admin />` component, so that it remains protected.